refactor(includes): logging-functions exclude wp_login action from logging

paulkirspuu opened the pull request:
GH repo version vs live is different, my proposed version is what would go live.

We should probably first update repo, recommit and go from there. For now I commited this so we can move fast with what's important.

paulkirspuu committed:
refactor(includes): logging-functions exclude wp_login action from logging

GH repo version vs live is different, my proposed version is what would go live.

We should probably first update repo, recommit and go from there. For now commited so we can move fast with what's important.

pawelkmpt commented:
As far as I see GH version is `3.7.0` and live site `1.4.11` so huge mismatch. I think we either have to create repo or branch for `1.4` . Otherwise this PR doesn't make sense.

lkraav commented:
Yes, all work must be based on 1.4 branch, which our fork already holds.

But indeed, warmpress copy should be re-checked vs live copy. Can't recall any reason for divergence off the top.

paulkirspuu committed:
refactor(includes): logging-functions stop empty meta logging

Note: GH repo version vs live is different, my proposed version is what matches live.

paulkirspuu commented:
Quite straightforward this one https://github.com/conversionxl/badgeos/pull/1/commits/4c9627b7c4586bbbbb55e3c15c1b17e472b7e5d7.

Don't see how it could be done without interfering plugin side, and since its deprecated anyway, no point in complicating things.

paulkirspuu commented:
I can deploy these 2 commits live now, I'd estimate they're harmless enough to do this on a Friday.

Could run DB cleanup in weekend then, or part of it.

pawelkmpt commented:
> Yes, all work must be based on 1.4 branch, which our fork already holds.

@lkraav already holds where? What did you mean? Repo tag?

> I can deploy these 2 commits live now, I'd estimate they're harmless enough to do this on a Friday.

@paulkirspuu did you deploy?

Could you post diff screenshot between updates and live site?

paulkirspuu commented:
Deployed now, per https://cxlworld.slack.com/archives/C01HXU5GGJG/p1705316582199729?thread_ts=1703154651.995509&cid=C01HXU5GGJG.

> Could you post diff screenshot between updates and live site?

My changes show what's on live vs what's here in this repo. Change between the two is minimal and irrelevant for small changes like these, but if we were to do something bigger in patching BadgeOS, we'd need to sync the two (99% certain we won't).

paulkirspuu commented:
Deployed now, per https://cxlworld.slack.com/archives/C01HXU5GGJG/p1705316582199729?thread_ts=1703154651.995509&cid=C01HXU5GGJG.

> Could you post diff screenshot between updates and live site?

![image](https://github.com/conversionxl/badgeos/assets/7458744/d94f3580-0fdd-48d1-8150-5b5831a71d24)

This is vs live.

Change in overall code between live vs repo is minimal and irrelevant for small changes like these, but if we were to do something bigger in patching BadgeOS, we'd need to sync the two (99% certain we won't).

pawelkmpt commented:
@paulkirspuu @lkraav I created branch `cxl-1.4` based on https://github.com/opencredit/badgeos/releases/tag/1.4.11 and using 5877d41a5c1a6dc19452bb25bf0f4654f26f7ea0 hash.

Changing base of this PR doesn't help as it carries over hundreds of commits (as it was branched off from `3.7.x`) so branch would need to be re-done.

Before it'd be good if @paulkirspuu compared `cxl-1.4` with the live site. If any differences present - commit them, then file updated PR.
